setup.cfg

```
@@ -28,4 +28,4 @@ replace = __version__ = "{new_version}"
 [flake8]
 ignore = E203, E231, W503, E722
 max-line-length = 79
-exclude = __init__.py,build,
+exclude = __init__.py,build,setup.py
```

See my suggestion, don't think this is needed any more

```suggestion
exclude = __init__.py,build
```